test/Sema/superfluously-public-imports.swift

```diff
@@ -139,6 +177,24 @@ public func publicFuncUsesPrivate() {
   let _: ClazzAlias
   // expected-remark @-1 {{type alias 'ClazzAlias' is imported via 'Aliases'}}
   // expected-remark @-2 2 {{typealias underlying type class 'Clazz' is imported via 'AliasesBase'}}
+
+  let x = ConformingType()
+  // expected-remark @-1 {{struct 'ConformingType' is imported via 'ConformanceBaseTypes'}}
+  // expected-remark @-2 {{initializer 'init()' is imported via 'ConformanceBaseTypes'}}
+  x.extFuncA() // expected-remark {{instance method 'extFuncA()' is imported via 'ExtensionA'}}
+  x.extFuncB() // expected-remark {{instance method 'extFuncB()' is imported via 'ExtensionB'}}
+}
+
+public struct StructUsingPropertyWrapper {
+  @MyPropertyWrapper(42) public var wrapped: Any // expected-remark 2 {{generic struct 'MyPropertyWrapper' is imported via 'PropertyWrapper'}}
+}
+
+extension PublicExtendedType { // expected-remark 2 {{struct 'PublicExtendedType' is imported via 'ExtendedDefinitionPublic'}}
+    public func foo() {}
+}
+
+extension NonPublicExtendedType {
```

Comment:
Are we supposed to see a remark here besides the warning in the import line or no?

Reply:
No remark here as this extension is internal because of its members. As long was just look at API, we basically get the remark or the warning, never both for the same imported module.